sub-rat1/microscopy/sub-rat1_sample-data1_SEM.json

```
@@ -0,0 +1,11 @@
+{
+    "PixelSize": 0.18,
+    "FovWidth": 230,
+    "FovHeight": 166,
+    "Species": "rat",
+    "BodyPart": "CSPINE",
+    "SampleFixation": "4% paraformaldehyde - 2% glutaraldehyde",
```

to avoid confusion with "minus"
```suggestion
    "SampleFixation": "4% paraformaldehyde, 2% glutaraldehyde",
```